docs/week15_scrum.md

```diff
@@ -0,0 +1,34 @@
+# Team Meetings
+
+Date: Jan 21, 2022
+
+Attendees: Raouf Tiliouine, Josh Cooper, Brian Zhou, Iwan Levin
+
+Absent:
+
+## 1. Project
+
+**Progress**
+
+- Fixed malformed buffer copy issue
+- Added documentation to key files (inline comments)
+- General code maintenance
+
+**Blockers + Surprises**
+
+N/A
+
+## 2. Learning Activity
+
+- Researched ODBC
+  - Explored benefits over undocumented Orthanc SDK functions
+  - Explored cost of implementation
+  - Explored performance and scaling
+
+## 3. Plan
+
+**Task for this week:**
+
+- Start implementing ODBC & crosswalk table
+- Start building frontend
```

Comment:
same thing, the tasks/activities are supposed to have names associated

Reply:
Added